### ISSUE TITLE
Some links in docs page are broken

### ISSUE THREAD

**Issue by hetsketch** [mabe02/lanterna#301]
This links in "Tutorials" section are 404: 
**Tutorial 2 - More Terminal functionality
Tutorial 3 - Using the Screen layer**

**Comment by mabe02** [comment]
Yep, this is in the process of merging the older (2.x) docs with the newer tutorials.

**Comment by avl42** [comment]
The link to "Tutorial 3" actually points to the Tutorial02.md - probably a little copypaste'o.

**Comment by mabe02** [comment]
Yep, that's a typo. Fixed that.
